Gemfile

```diff
@@ -10,6 +10,11 @@ group :examples do
   #gem 'unicode' if (Gem::Version.new RUBY_VERSION) < (Gem::Version.new '2.4.0')
 end
 
+group :test, :development do
+  gem 'rspec'
```

Comment:
For the time being, development dependencies should be listed in the gemspec (where you will find rspec already listed).